pallets/creditcoin/src/migrations/v4.rs

```diff
@@ -20,16 +20,13 @@ pub struct Address<AccountId> {
 	pub owner: AccountId,
 }
 
-generate_storage_alias!(
-	Creditcoin,
-	Addresses<T: Config> => Map<(Blake2_128Concat, AddressId<T::Hash>), Address<T::AccountId>>
-);
+#[storage_alias]
+type Addresses<T: Config> =
+	StorageMap<crate::Pallet<T>, Blake2_128Concat, AddressId<HashOf<T>>, Address<AccountIdOf<T>>>;
```

Comment:
Why not use the new macros for consistency?

Reply:
The idea behind `storage_macro` was that we have around 15 instances like
```rust
// In migration
#[frame_support::storage_alias]
type AskOrders<T: crate::Config> = StorageDoubleMap<
	crate::Pallet<T>,
	Twox64Concat,
	BlockNumberOf<T>,
	Identity,
	HashOf<T>,
	AskOrder<AccountIdOf<T>, BlockNumberOf<T>, HashOf<T>>,
>;

// In migration tests
#[frame_support::storage_alias]
type AskOrders<T: crate::Config> = StorageDoubleMap<
	crate::Pallet<T>,
	Twox64Concat,
	BlockNumberOf<T>,
	Identity,
	HashOf<T>,
	OldAskOrder<AccountIdOf<T>, BlockNumberOf<T>, HashOf<T>, MomentOf<T>>,
>;
```

Where we make 2 storage aliases, where the only difference is in the final type (`AskOrder`/`OldAskOrder`). The same duplication happens for bid orders, deal orders, etc. So `storage_macro` pretty much just takes everything except for the last type parameter (`AskOrder` e.g.) and gives you back a new macro (`ask_orders_storage`) that just takes the final parameter and expands to the appropriate alias.

Since there isn't that duplication here, there isn't any benefit over just `storage_alias`.